docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md

```diff
@@ -121,37 +121,58 @@ for the `geoLocation` of the `WindTurbine` and `WindFarm`.
 
 ## Updating the Spreadsheet.
 
-Olav copies over all the classes and properties from the enterprise model to the solution model. He then removes
-the rows that are not needed for the new solution model. The spreadsheet will now look as follows
-(details of other columns excluded on purpose):
-
-| Class       | Property       | Value Type  |
-|-------------|----------------|-------------|
-| WindTurbine | name           | string      |
-| WindTurbine | type           | string      |
-| WindTurbine | activePower    | timeseries  |
-| WindTurbine | geoLocation    | Point       |
-| WindTurbine | manufacturer   | string      |
-| WindTurbine | ratedPower     | float       |
-| WindTurbine | hubHeight      | float       |
-| WindTurbine | lifeExpectancy | integer     |
-|             |                |             |
-| WindFarm    | name           | string      |
-| WindFarm    | geoLocation    | Polygon     |
-| WindFarm    | windTurbines   | WindTurbine |
-|             |                |             |
-| GeoLocation | name           | string      |
-|             |                |             |
-| Point       | latitude       | float       |
-| Point       | longitude      | float       |
-|             |                |             |
-| Polygon     | point          | Point       |
+Olav copies over all the properties from the enterprise model to the solution model. He then removes
+the rows that are not needed for the new solution model. The `Properties` spreadsheet will now look as follows
+(only the most relevant columns shown here):
```

Comment:
Great! I think I will need to update kg acquisition tutorila, and reduce the size of tables, since currently they are too long!